src/common/navigation/MainToolbar.tsx

```diff
@@ -28,7 +30,7 @@ const MainToolbar: React.FC = () => {
           </>
         ) : (
           <>
-            <Button href={'/login'} sx={{ marginLeft: 'auto', borderRadius: 'sm', color: '#CF7F49' }}>Log in</Button>
+            <Button href={'/api/auth/signin'} sx={{ marginLeft: 'auto', borderRadius: 'sm', color: '#CF7F49' }}>Log in</Button>
```

Comment:
Is this route supposed to still be to the api instead of to a page view?

Reply:
That was an attempt to get NextAuth to recognize me, it could be either that or /signin and work fine

Reply:
Okay, let's probably switch that back. I'd rather stick to using page routes except in specific cases that require api routes

Reply:
(It probably doesn't really matter... but I don't want to dig through the docs figuring out for sure, so that's why I'm leaning just stick to pages haha)